src/t_string.c

```diff
@@ -109,6 +109,28 @@ void setGenericCommand(client *c,
         return;
     }
 
+    /* If the milliseconds have expired, then we don't need to set it into the
```

Comment:
This sentence didn't make sense immediately. Could we add back tick to the variable name like `milliseconds`